### PR TITLE
[Feature] Table texts without quality evaluations

### PR DESCRIPTION
**Pull request by carlosvillu**
This change is [Reviewable](https://reviewable.io/reviews/carlosvillu/uabtexthandler/15)

### CODE REVIEW

File: src/components/TableTextsWithoutQualityEvaluations/component.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  async componentDidMount() {
+    const {setStateEvaluations, domain} = this.props
+    setStateEvaluations(
+      await domain
+        .get('get_stats_texts_without_quality_evaluations_use_case')
+        .execute()
+    )
+  }
+
+  componentWillUnmount() {}
```
Comment (davecarter) [inline src/components/TableTextsWithoutQualityEvaluations/component.js]: empty methid

File: src/components/TableTextsWithoutQualityEvaluations/component.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+            }
+          ]}
+          defaultPageSize={20}
+          className="-striped -highlight"
+          style={{
+            height: '80vh'
+          }}
+        />
+        <br />
+        <div style={{textAlign: 'center'}}>
```
Comment (davecarter) [inline src/components/TableTextsWithoutQualityEvaluations/component.js]:
```suggestion
        <div className='.inlineCenter'>
```

File: src/domain/evaluations/Repositories/FireBaseEvaluationsRepository.js [hunk 3]
```diff
@@ -58,4 +60,14 @@ export default class FireBaseEvaluationsRepository extends EvaluationsRepository
       [type.value()]: (entry[type.value()] || 0) + 1
     })
   }
+
+  async without() {
+    const refsManager = this._config.get('refsManager')
+    const textsRef = refsManager.ref({path: '/texts'})
+    const texts = (await textsRef.once('value')).val() || {}
+    const textEntities = Object.entries(texts)
```
Comment (davecarter) [inline src/domain/evaluations/Repositories/FireBaseEvaluationsRepository.js]: this should be in a mapper

File: src/components/TableTextsWithoutQualityEvaluations/component.js (hunk 2 above)
Comment (carlosvillu) [reply src/components/TableTextsWithoutQualityEvaluations/component.js]: I didnt change that!

File: src/domain/evaluations/Repositories/FireBaseEvaluationsRepository.js (hunk 3 above)
Comment (carlosvillu) [reply src/domain/evaluations/Repositories/FireBaseEvaluationsRepository.js]: You are right. Good one!

**Pull request merged**